python/ray/autoscaler/sdk.py

```diff
@@ -185,19 +186,20 @@ def request_resources(num_cpus=None, bundles=None):
 
 
 @contextmanager
-def _as_config_file(cluster_config: Union[dict, str]):
+def _as_config_file(cluster_config: Union[dict, str]) -> Iterator[str]:
     if isinstance(cluster_config, dict):
         tmp = tempfile.NamedTemporaryFile("w", prefix="autoscaler-sdk-tmp-")
         tmp.write(json.dumps(cluster_config))
         tmp.flush()
-        cluster_config = tmp.name
-    if not os.path.exists(cluster_config):
-        raise ValueError("Cluster config not found {}".format(cluster_config))
-    yield cluster_config
+        cluster_config_file = tmp.name
```

Comment:
I think this should remain as before. We are concerned about a user passing in a `str` to `cluster_config` and that string not representing a file.

Reply:
You're right, thanks! 
The change didn't make sense.